### ISSUE TITLE
[Bug] Return to blank GUI when Chia left unattended. 

### ISSUE THREAD

**Issue by OldButtKicking** [Chia-Network/chia-blockchain#10464]
### What happened?

This has happened to me numerous times. I leave Chia running fine for a week or so and when I return and bring up the GUI it is a blank white form with only the menu visible. looking at the log it appears to be farming correctly and this must just be a GUI issue.  The only option is to shut it down and restart it
![Screenshot 2022-02-27 083259](https://user-images.githubusercontent.com/58090640/155890929-a117526a-d53e-486c-af8c-5349326cbeba.png)
.

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
most recent error that looks ui related

2022-02-27T01:19:52.098 daemon __main__                   : INFO     Websocket exception. Closing websocket with wallet_ui code = 1006 (connection closed abnormally [internal]), no reason Traceback (most recent call last):
  File "asyncio\windows_events.py", line 457, in finish_recv
OSError: [WinError 64] The specified network name is no longer available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "websockets\protocol.py", line 827, in transfer_data
  File "websockets\protocol.py", line 895, in read_message
  File "websockets\protocol.py", line 971, in read_data_frame
  File "websockets\protocol.py", line 1047, in read_frame
  File "websockets\framing.py", line 105, in read
  File "asyncio\streams.py", line 723, in readexactly
  File "asyncio\streams.py", line 517, in _wait_for_data
  File "asyncio\proactor_events.py", line 280, in _loop_reading
  File "asyncio\windows_events.py", line 812, in _poll
  File "asyncio\windows_events.py", line 461, in finish_recv
ConnectionResetError: [WinError 64] The specified network name is no longer available

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "Lib\site-packages\chia\daemon\server.py", line 195, in safe_handle
  File "websockets\protocol.py", line 439, in __aiter__
  File "websockets\protocol.py", line 509, in recv
  File "websockets\protocol.py", line 803, in ensure_open
websockets.exceptions.ConnectionClosedError: code = 1006 (connection closed abnormally [internal]), no reason
```

**Comment by OldButtKicking** [comment]
Actually I just discovered the view reload option on the menu and this also fixed the problem.

**Comment by esaung** [comment]
also please try with 1.3 beta and see if the issue is still happening there for you.

**Comment by OldButtKicking** [comment]
ok but this has not happened on demand usually only appears after 1 to 2 weeks of the ui running.

**Comment by emlowe** [comment]
We believe this is resolved in 1.3.4 - feel free to re-open if seen again

**Comment by josinaldobarbosa** [comment]
@emlowe this is happening in 1.3.5
